pkg/scanners/trivy/helpers.go

```diff
@@ -27,7 +27,7 @@ func loadConfig(filename string) (Config, error) {
 		return cfg, err
 	}
 
-	var eraserConfig eraserv1alpha1.EraserConfig
+	var eraserConfig unversioned.EraserConfig
```

Comment:
This should be addressed in a separate PR, but we will want to manage a versioned schema for the scanner config as well.